### PR TITLE
AVRO-3906: Add .NET 8.0 support

### PR DESCRIPTION
**Pull request by zcsizmadia**
## What is the purpose of the change

*Add .NET 8.0 support (AVRO-3906)*

## Verifying this change

This change is already covered by existing tests.

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by zcsizmadia** [comment]
It seems like there is an issue with the C# ARM64 build if the XZ library package Joveler.Compression.XZ is bumped to the latest. The latest package works on my dev environment and succeeds with the standard github worklows, except on ARM64. Added comment about it in the `versions.props`.

File: lang/csharp/versions.props
Comment (martin-g) [inline lang/csharp/versions.props:39]: Is it worth it to report the issue to the Joveler developers ?

**Review by martin-g — APPROVED** [review]

File: lang/csharp/versions.props
Comment (zcsizmadia) [reply lang/csharp/versions.props:39]: Yes, I will. I just wanted to reproduce and investigate before I do that.

**Comment by zcsizmadia** [comment]
@martin-g This PR fixes AVRO-3909. I wanted to wait for people give some suggestion before I merge, however this PR is fairly simple, so I can go ahead and merge it now to resolve the Newtonsoft vulnerability warning.

**Comment by zcsizmadia** [comment]
Fyi, I created #2523 back in September asa preparation for the NET 8 analyzer  warning about it, however it wont be needed any more, I will reject that PR once this is merged

**Comment by zcsizmadia** [comment]
Merged. @martin-g if you rebase those failing PRs with main and it should be ok now.

**Comment by stfnzl** [comment]
Nice work! Definitely helps!
Is there a plan to release this on NuGet as well?

**Pull request merged**
